_topics/en/multisignature.md

```diff
@@ -0,0 +1,108 @@
+  requires security assumptions beyond those normally required for
+  Bitcoin-style ECDSA signatures.  It's easier to create multisignatures
+  for [schnorr signatures][topic schnorr signatures], and several
+  algorithms for that are known, with [MuSig][topic musig] having been
+  specifically created for the needs of Bitcoin users.
+
+  **Terminology:** multisignatures should not be confused with
+  *multisig*, the use of public keys and signatures with Bitcoin's
+  `OP_CHECKMULTISIG` and `OP_CHECKMULTISIGVERIFY` opcodes (and the
+  `OP_CHECKSIGADD` opcode proposed for [tapscript][topic tapscript]).
+  The following table summarizes the differences between *multisignature*
+  and several related terms.
+
+  | Term | Keys | Messages<br>(e.g. tx inputs) | Signatures | Signers required | Notes |
+  |-|-|-|-|-|
+  | Multisig | `m` | `1` | `k` where `k<=m` | `k` | Uses `OP_CHECK*SIG` opcodes |
+  | Multisignature | `m` | `1` | 1 | `m` | Indistinguishable onchain from single-sig |
+  | Threshold signature | `m` | `1` | `1` | `k` where `k<=m` | Indistinguishable onchain from single-sig |
+  | Signature aggregation | `m` | `n` | 1 | `0` or `m`<br>(varies by scheme) | Proposed to allow a tx with multiple inputs to have only one signature, among other uses |
+
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+#primary_sources:
+#    - title: Test
+#    - title: Example
+#      link: https://example.com
+
+## ##################################################################### ##
+## NOTE: don't include things specific to MuSig; put those in the MuSig  ##
+## topic, which we link to in the see_also section                       ##
+## ##################################################################### ##
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: Discussion of multisignatures and threshold signatures
+    url: /en/newsletters/2020/07/01/#schnorr-signatures-and-multisignatures
+
+  - title: Mitigating power analysis attacks, including against multisignature schemes
+    url: /en/newsletters/2020/04/01/#mitigating-differential-power-analysis-in-schnorr-signatures
+
+  - title: Discussion about nested and composable multisignature schemes
+    url: /en/newsletters/2019/12/04/#continued-schnorr-taproot-discussion
+
+  - title: "BIPs #876 assigns BIP340 to multisignature compatible scheme"
+    url: /en/newsletters/2020/01/29/#bip340
+
+  - title: ECDSA multisignatures for scriptless Lightning Network payment channels
+    url: /en/newsletters/2018/10/09/#multiparty-ecdsa-for-scriptless-lightning-network-payment-channels
+
+  - title: Alternative to ECDSA multisignatures for signature adaptors
+    url: /en/newsletters/2020/04/08/#work-on-ptlcs-for-ln-using-simplified-ecdsa-adaptor-signatures
+
+  - title: Warning about using 160-bit addresses for naïve multiparty multisignatures
```

Comment:
A quick search shows 5 instances of _naive_ and 0 of _naïve_ in our newsletters.